docs/installation/configuration-files.rst

```diff
@@ -41,6 +41,19 @@ logging
          handlers: ["myCustomHandler"]
          level: DEBUG
 
+content
+^^^^^^^
+
+The configuration for the content serving application.
+
+web_server
+  Defines the type of web server that is running the content application.
+  When set to `django`, the content is streamed.
+  When set to `apache`, the `X-SENDFILE header` is injected which delegates
```

Comment:
s/`X-SENDFILE header`/`X-SENDFILE` header/

Reply:
Sorry, I don't see the difference?

Reply:
haha it's so nitpicky- header is inside the backticks instead of outside.

Reply:
:-) fixed.